cg/exc.py

```diff
@@ -127,8 +126,8 @@ class HousekeeperFileMissingError(CgError):
     Exception raised when a file is missing in Housekeeper.
     """
 
-    def __init__(self, message, errors=None):
-        self.message = message
+    def __init__(self, message: str = "", errors=None):
```

Comment:
```suggestion
    def __init__(self, message: str = EMPTY_STRING, errors=None):
```
Matter of style - what do you think of using a named constant instead. It does read better.

We have some in cg/constants/symbols.py

Reply:
I agree for those that are currently in `cg/constants/symbols.py` but for an empty string i quite like `""`. I feel like I would be more confused by `EMPTY_STRING`. I don't have strong opinion though, so feel free to overrule me 😉

Reply:
Ok, let's keep it as it is. 👍